Codecs/DAG-PB.md

```diff
@@ -32,24 +32,69 @@ message PBNode {
 ```
 
 The objects link names are specified in the 'Name' field of the PBLink object.
-This is non-standard and means that the 'Tsize' attribute of a link, and
-neither the 'Data' field nor the 'Links' field of the PBNode are available
-through the ipld query language. 
+All links in an object must either be blank or unique within the object.
```

Comment:
A link itself can be blank? Or PBNode.Links can be blank? If the former, does that mean that "optional" in all the PBLink fields are truly optional? In the JS implementation they're not, a CID is required and links created from within JS will have a default name of empty-string if none is supplied.

Reply:
Sorry, that should have said "link names".